tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
+    "target": "es6",
```

Comment:
Why also change the target? If you change the target, then consumers will have to transpile. If you leave the target as `es5` and change the module type to `es6`, webpack will be able to do the right thing.